builder/src/bin/permissionless-builder.rs

```diff
@@ -161,7 +151,7 @@ async fn run<V: Versions>(
         opt.hotshot_event_streaming_url,
         builder_server_url,
         api_response_timeout_duration,
-        buffer_view_num_count,
+        ChainConfig::default().max_block_size.into(),
```

Comment:
Uhh @shenkeyao why are we defaulting this? This should come from the actual chain config, e.g. `instance_state.chain_config.max_block_size`

Reply:
Good catch! Using `instance_state.chain_config` also allows us to simplify the function by removing this argument, since we already have `instance_state` as an arg.